src/Microsoft.Build.Sql.Templates/sqlproject/.template.config/template.json

```diff
@@ -15,7 +15,25 @@
     },
     "sourceName": "SqlProject1",
     "preferNameDirectory": true,
+    "sources": [
+        {
+            "source": "./",
+            "target": "./",
+            "include": ["SqlProject1.sqlproj", "README.md"]
+        },
+        {
+            "source": "./",
+            "target": "./",
+            "include": [ ".gitignore"],
+            "condition": "(gitignore)"
+        }
+    ],
     "symbols": {
+        "gitignore": {
```

Comment:
I don't see the shorthand configured anywhere, is it just by default the first letter of the parameter?

Reply:
It is, I wasn't able to find any more info on the symbol command line flag config in https://github.com/dotnet/templating/wiki/Reference-for-template.json